packages/test/test-version-utils/src/versionUtils.ts

```diff
@@ -40,6 +40,12 @@ interface InstalledJson {
 	installed: string[];
 }
 
+let cachedInstalledJson: InstalledJson | undefined;
```

Comment:
What do you think of this whole caching thing? I was thinking of turning this whole thing into a whole object, but that might be a refactoring for later.

Reply:
@Abe27342

Reply:
seems ok to me. We do kind of now have split-brain here around how we manage the state of currently installed packages, but i don't think the before state is particularly elegant and it's all implementation details, so easy enough to change when we have time to improve further.